spec/lib/ruby_inspector_spec.rb

```diff
@@ -6,29 +6,53 @@
   end
 
   describe '.enable' do
-    it 'sends an initialization message to the ruby inspector server' do
-      allow(described_class).to receive(:connect)
-      expect(described_class).to receive(:send_info).with(
-        method: 'RubyInspector.initialize',
-        params: {
-          name: 'test app',
-          type: :ruby,
-          description: 'test app description'
-        }
-      )
-      described_class.enable('test app', 'test app description')
+    let(:socket) { double('socket') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.